platform/src/main/java/org/stellar/anchor/platform/configurator/ConfigHelper.java

```diff
@@ -44,7 +46,7 @@ public static ConfigMap loadConfigFromEnv(int latestVersion) throws InvalidConfi
     // Determine the version.
     // if system.env['version'] is not defined, use the latest version.
     int version;
-    String strVersion = ConfigEnvironment.getenv("version");
+    String strVersion = ConfigEnvironment.getenv("VERSION");
```

Comment:
Can we also document what format the ConfigEnvironment is storing (posix format, I guess) and what format the ConfigMap is storing? This will help us to remember in the future.